cloud/shared/bin/pgadmin.py

```diff
@@ -0,0 +1,19 @@
+import subprocess
+import importlib
+import os
+from typing import List
+
+from cloud.shared.bin.lib.config_loader import ConfigLoader
+
+
+def run(config: ConfigLoader, params: List[str]):
```

Comment:
To launch pgadmin the human admin must provide `--tag=...` option, right? Which means they need to use exact version of the app that is currently deployed. Otherwise they will accidentally re-deploy civiform which is pretty undesirable. Can we detect it somehow and prevent? Initially I thought we should just forbid using "--tag=latest" with pgadmin but then remembered that we want it to work on staging as well.

Maybe we can extract previous version from terraform state? Alternatively we could try hacks like reading currently deployed version from civiform webpage html.

Reply:
What do you think about the approach in https://github.com/civiform/civiform-deploy/pull/30?  It avoids needing to dynamically detect anything

Reply:
Looks good!